management/src/clusterm/manager/commission_event.go

```diff
@@ -17,17 +17,19 @@ type commissionEvent struct {
 	mgr       *Manager
 	nodeNames []string
 	extraVars string
+	hostGroup string
 
 	_hosts  configuration.SubsysHosts
 	_enodes map[string]*node
 }
 
 // newCommissionEvent creates and returns commissionEvent
-func newCommissionEvent(mgr *Manager, nodeNames []string, extraVars string) *commissionEvent {
+func newCommissionEvent(mgr *Manager, nodeNames []string, flags ActionFlags) *commissionEvent {
```

Comment:
commission event can take `hostgroup` as an explicit argument. Or we can replace `nodeNames []string` with `*APIRequest` itself. `extra-vars` will also soon be available in that structure once #99 is fixed.